app/src/main/res/menu/bottom_menu_canvas.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/add_text"
+        android:icon="@drawable/baseline_article_24"
+        android:title="Text" />
+    <item
+        android:id="@+id/sharedFragment"
```

Comment:
use snake case, same with below

Reply:
These are placeholders, will be gotten rid of

Reply:
the ids?

Reply:
yes they change in the next pr lol